docs/operator-guide/add-ons-overview.md

```diff
@@ -14,88 +14,137 @@ In a nutshell, Add-Ons are separate Helm Charts that can be installed with just
+    │   │   │   │   ├── aws-efs-csi-driver.yaml
+    │   │   │   │   ├── ...
+    │   │   │   ├── Chart.yaml
+    │   │   │   ├── README.md
+    │   │   │   └── values.yaml
+    │   │   └── bootstrap-addons.yaml
+    │   └── prod
+    ├── ...
     ```
+    </details>
 
-* **add-ons** - the directory that contains Helm charts of the applications that can be integrated with KubeRocketCI using Add-Ons.
+* **argo-cd** - The directory containing the Helm chart for deploying the main Argo CD instance and prepared project templates.
 
-* **chart** - the directory that contains Helm charts with application templates that will be used to create custom resources called Applications for Argo CD.
+* **clusters** - A directory for organizing configurations and Helm charts specific to individual clusters. Each subdirectory corresponds to a particular cluster environment (e.g., core, prod, or dev).
+
+* **addons** - The directory containing subdirectories with Helm charts for applications that can be integrated with KubeRocketCI.
+
+* **apps** - Contains application `templates` directory used to create Argo CD application. The deployment of these applications is managed by modifying the `apps/values.yaml` file, where you can enable or disable specific applications.
+
+* **bootstrap-addons.yaml** - The manifest file defining the app of apps application responsible for deploying all enabled application to the cluster. This manifest is applied manually.
 
 ## Enable KubeRocketCI Add-Ons
 
 To enable Add-Ons, it is necessary to have the configured Argo CD, and connect and synchronize the forked repository. To do this, follow the guidelines below:
 
-1. Fork the Add-Ons repository to your personal account.
+1. Create a private copy of the [Add-Ons repository](https://github.com/epam/edp-cluster-add-ons) in your personal Git account.
 
-2. Provide the parameter values for the values.yaml files of the desired Add-Ons you are going to install.
+2. Align **repoUrl** in [/clusters/core/apps/values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/apps/values.yaml#L14), [/clusters/core/bootstrap-addons.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/bootstrap-addons.yaml#L16) and **repoSource** in the [/argo-cd/templates/appProjectCore.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/argo-cd/templates/appProjectCore.yaml#L36), [/argo-cd/templates/appProjectCore.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/argo-cd/templates/appProjectKRCI.yaml#L55) files of the repository, specify the SSH URL of your private repository for github replace `kuberocketci` to your `<github_account_name>`.
 
-3. Navigate to **Argo CD** -> **Settings** -> **Repositories**. Connect your forked repository where you have the values.yaml files changed by clicking the **+ CONNECT REPO** button:
+3. Clone forked repository and install prepared **Argo CD** helm chart from `/argo-cd` folder.
```

Comment:
in point 2 we align some parameters here, maybe we need to remove 'clone' tips from here or add in point 2?